admin_app/src/app/_document.tsx

```diff
@@ -0,0 +1,58 @@
+import Document, {
```

Comment:
Adding this document is what fixed the font bug by making sure the Server Side Rendering is the same as the Client Side Rendering! It raised a warning in the console but I will fix it in next PR

Reply:
Alright. I will remove this fix and put the task in the backlog.